usaspending_api/agencies/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+from django.utils.translation import ugettext_lazy as _
```

Comment:
`_` is used by convention to represent localization functions in Python.

https://docs.python.org/3.5/library/gettext.html#gettext.gettext

Additionally, it's how it's used in the documentation and throughout Django.

https://docs.djangoproject.com/en/1.11/topics/i18n/translation/

Honestly, I'm not a fan myself since `_` is _also_ used by convention to represent throwaway values.

Reply:
I'll allow it, but I hate it. It doesn't make any sense